assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -46,53 +46,75 @@ function APIProvider({ children }) {
     [stories]
   );
 
-  const saveStoryById = useCallback(
-    /**
-     * Fire REST API call to save story.
-     *
-     * @param {import('../../types').Story} story Story object.
-     * @return {Promise} Return apiFetch promise.
-     */
-    ({
-      storyId,
+  const getStorySaveData = ({
+    title,
+    status,
+    pages,
+    author,
+    slug,
+    date,
+    modified,
+    content,
+    excerpt,
+    featuredMedia,
+    password,
+    stylePresets,
+    publisherLogo,
+    autoAdvance,
+    defaultPageDuration,
```

Comment:
There must be a better way to do this? Maybe we can restructure it into a few groups of variables? Maybe we can `...rest` a lot of them - maybe combined with `objectPick` or `objectWithout`? We surely don't need to iterate every single one here?

Reply:
After your comment, tried it out with `rest` and it failed -- which lead to finding an unnoticed incorrect `saveStory` call. Fixed in ecccaed